AdblockVPNGuide.md

```diff
@@ -285,7 +285,7 @@
 * ⭐ **[Mullvad Browser](https://mullvad.net/en/browser)** - Tor Browser Fork (w/o Tor network)
 * ⭐ **[Librewolf](https://librewolf.net/)**- Pre-tweaked Firefox Fork / **[Use Auto-Updater](https://github.com/ltguillaume/librewolf-winupdater)**
 * ⭐ **[arkenfox](https://github.com/arkenfox/user.js)**- Firefox Privacy Tweak / [Video](https://youtu.be/F7-bW2y6lcI)
-* [Brave](https://brave.com/) - Chromium Based Browser
+* [Brave](https://brave.com/) - Chromium Based Browser / [Warning](https://rentry.co/brave-trash)

```

Comment:
Dont really feel this "Warning" is needed. Its just stupid (and no I dont use brave)